tests/oeedger8r/enc/CMakeLists.txt

```diff
@@ -47,3 +48,7 @@ if (CMAKE_CXX_COMPILER_ID MATCHES GNU OR CMAKE_CXX_COMPILER_ID MATCHES Clang OR
 endif()
 
 target_include_directories(edl_enc PRIVATE ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/..)
+if (NOT WIN32)
```

Comment:
For enclaves, is there some issue with enabling this on Windows?

Reply:
Yeah it doesn't like the compiler flag -Werror; do you know the right way to do it on Windows? I didn't see any gain spending time on it since we have the check on Linux.

Reply:
Based on this: https://oe-jenkins.eastus.cloudapp.azure.com/blue/organizations/jenkins/Bors/detail/trying/839/pipeline/
the error is happening on the host.

We should not be passing in the flag for the host. But we should be passing in the flag for enclaves.

Reply:
I mean there's no reason not to enable strict-aliasing for of our code.

Reply:
As this code will disappear shortly with the resolution of #1717 (which this PR is blocking), I am going to punt this issue. We will have strict-aliasing enabled for all of our code in a Windows and Linux compatible manner.